### PR TITLE
EVG-6267 UI fixes, make periodic builds discoverable

### PR DESCRIPTION
**Pull request by john-m-liu**
example build: https://evergreen-staging.corp.mongodb.com/version/5d3615e79dbe32172d795887

### CODE REVIEW

File: public/static/partials/github_commit_panel.html [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
     </div>
     <div class="commit-message-author">
-      <span class="semi-muted">
+      <span ng-if="commit.author" class="semi-muted">
         [[commit.author]]
         ([[commit.author_email]])
       </span>
       <span class="muted">
-        authored at [[(commit.ingest_time || commit.create_time) | convertDateToUserTimezone:timezone:"MMM D, YYYY h:mm:ss a"]]
+        [[ commit.author ? 'authored' : 'Created' ]] at [[(commit.ingest_time || commit.create_time) | convertDateToUserTimezone:timezone:"MMM D, YYYY h:mm:ss a"]]
```
Comment (bsamek) [inline public/static/partials/github_commit_panel.html]: nit: consistent capitalization 'authored', so 'created'

Comment (john-m-liu) [reply public/static/partials/github_commit_panel.html]: Created is the start of the sentence here, whereas a name starts the sentence in the other scenario

**Comment by john-m-liu** [comment]
My thought was to make them discoverable via notifications, though I'm open to other ideas

**Comment by bsamek** [comment]
> My thought was to make them discoverable via notifications, though I'm open to other ideas

I think that's reasonable, though it might make sense to check with James first, in case you end up building something far away from what he wants.

**Pull request merged**